ui/src/pages/Admin/AllStories/reducer.ts

```diff
@@ -449,7 +456,7 @@ const requestSearchAndFilter = (newState: State) => {
     );
   } else if (newState.tabValue === 2) {
     newState.changedVisibility = requestSearchAndFilterHelper(
-      newState.changedVisibilityFilter,
+      newState.changedVisibility,
```

Comment:
This makes the search and filter not work properly. @hanlinc27 Was this change to fix the other issue?